devtools-ci-index.yaml

```diff
@@ -556,6 +556,10 @@
                 - openshift-connector
     triggers:
         - github
+        - github-pull-request:
```

Comment:
We should try/test the new job is working *before* this PR get merged. 

Anyway the new verification job will still be triggered by any commit. So basically whenever something get merged on either redhat-developer/rh-che `master` branch or eclipse/che `openshift-connector` branch. So if we want to verify it after merge we could still use these triggers.

I don't get why you think we can't verify it without the `github-pull-request` trigger. Am I missing something?